### DISCUSSION TITLE
宏達電VR部門傳赴海外掛牌！下一隻新創獨角獸在路上了？

### DISCUSSION THREAD

**Discussion by peggy0329** [1101-GS4719-C/Python-Community-GS4719-C#876] [General]
近期傳出宏達電將分割VR部門至海外掛牌上市，從新聞中我們可以得知過去幾年宏達電致力於發展VR相關產品，除了在 2021 年的企業轉型組織重組計畫中，創建七大事業部，為培育出獨角獸，宏達電也在台北、北京、深圳、舊金山、以色列等地成立 Vive X 加速器，Vive X 加速器僅成立一年多，就成為全球最活躍的虛擬實境，並已選入近 70 個全球新創團隊。

我對於宏達電旗下之VR部門的**前景是看好的**，主要有幾個原因：
1. 若旗下的VR事業赴美掛牌上市的話，有利於募得資金，同時也可以藉此擴大品牌知名度與影響力
2. 宏達電已經在手機市場失利了，在沒有退路的情況下，它不得不把已投入大量資金的VR事業發展好
3. 宏達電除了內部組織有所調整，也有積極參與相關大型組織，如：Meta、微軟、高通等企業組成的「元宇宙標準論壇」

不知道大家怎麼看呢？

資料來源：https://www.bnext.com.tw/article/73063/htc-vr

**Comment by CPCpyt** [comment]
我覺得的確近期的活動非常頻繁，
可以看出管理階層的願景以及決心，
希望在日後能構成為第二個護國神山！
